### ISSUE TITLE
Possibility to ignore videos from streaming services

### ISSUE THREAD

**Issue by xoniq** [Fallenbagel/jellyseerr#206]
### Description

I have like Netflix, Disney+, HBO and stuff, and I often find myself downloading something existing on services which isn’t very useful.

would be handy if we can ignore videos being on there.

### Desired Behavior

As mentioned in the description.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by rh535** [comment]
This would be great!

**Comment by Yankees4life** [comment]
Cosign on this feature too.

**Comment by dominikvpb** [comment]
It would be a great feature! Not sure how complicated it would be concerning the regional/country specific differences in e.g. Netflix. Because depending on where your Netflix subscription is the catalogue can be different.

**Comment by xoniq** [comment]
> It would be a great feature! Not sure how complicated it would be concerning the regional/country specific differences in e.g. Netflix. Because depending on where your Netflix subscription is the catalogue can be different.

Shouldn't be difficult at all, since the app already shows at what services its able to stream. 

Then it's just a matter of checking which services you own, and filter those results away.

**Comment by Fallenbagel** [comment]
This should be partially fixed in the next release (available on develop)

The reason why I say partially is because, filters were added with a whole movies/series tab where you can filter specific streaming services and such like so

![image](https://user-images.githubusercontent.com/98979876/215229552-2bf03df5-8b9a-45f6-87b6-a1bee0e07396.png)

**Comment by gmcouto** [comment]
@Fallenbagel yeah, the only problem is that it would be very useful to have a "negative" version of that filter. I only pay for Netflix, it would be super useful to see content that is anywhere but on Netflix.

**Comment by cmerkle** [comment]
Any updates on this? This would be an extremely useful feature, as I pay for Netflix, Max, Disney+, etc, and I only want to add movies to Jellyfin that don't exist on those platforms. It would just be a negative on the already existing streaming service filter.

**Comment by Fallenbagel** [comment]
https://www.themoviedb.org/talk/6615ba998ee0a90165a497a1

There's this issue.

**Comment by cfr2305** [comment]
I really would appreciate this feature. Are there any news about this?
In the meantime I was looking into [Elsewherr](https://github.com/Adman1020/Elsewherr), but going back and forth to Radarr makes no sense ;)

**Comment by geneh0** [comment]
HI! I randomly stumbled upon this and it looks like the TMDB endpoint has just been added. Does that unblock this feature request? Thanks!

**Comment by ThermicWaffle** [comment]
> https://www.themoviedb.org/talk/6615ba998ee0a90165a497a1
> 
> There's this issue.

This blocker has been resolved. Any updates on adding the feature?

**Comment by Tharic99** [comment]
Any update on this request? #2533 looks like it added this but it was rejected because it was using too much AI code.

**Comment by YaibaToKen** [comment]
Hi, any news on this?

**Comment by xoniq** [comment]
> Hi, any news on this?

After 4 years? I guess it's impossible. Harder to do than sending people to the moon.

**Comment by fallenbagel** [comment]
> Hi, any news on this?

Still open, still planned.

> After 4 years? I guess it's impossible. Harder to do than sending people to the moon.

Sure, let us just put everything else going on in our lives aside and fast-track this for you. Since it's free software you paid nothing for, we're volunteering time out of our lives to provide you with something you apparently feel entitled to. If you need it sooner, PRs are welcome.

**Comment by xoniq** [comment]
> > Hi, any news on this?
> 
> Still open, still planned.
> 
> > After 4 years? I guess it's impossible. Harder to do than sending people to the moon.
> 
> Sure, let us just put everything else going on in our lives aside and fast-track this for you. Since it's free software you paid nothing for, we're volunteering time out of our lives to provide you with something you apparently feel entitled to. If you need it sooner, PRs are welcome.

I could and have thought about it, but if I look at the PR's, they also stay open since forever so why bother creating a PR which end up under the pile.

**Comment by fallenbagel** [comment]
> > > Hi, any news on this?
> > 
> > Still open, still planned.
> > 
> > > After 4 years? I guess it's impossible. Harder to do than sending people to the moon.
> > 
> > Sure, let us just put everything else going on in our lives aside and fast-track this for you. Since it's free software you paid nothing for, we're volunteering time out of our lives to provide you with something you apparently feel entitled to. If you need it sooner, PRs are welcome.
> 
> I could and have thought about it, but if I look at the PR's, they also stay open since forever so why bother creating a PR which end up under the pile.

That's simply a huge lie. We release multiple features and bug fixes in every release, and we're currently at v3.4.1. Where do you think those come from? PRs. PRs get reviewed and merged almost every week.

The fact that this particular feature hasn't been prioritized doesn't mean nothing gets merged, nor does it mean we're sitting on our hands. You can literally see multiple commits in the git history. Almost every day/week/month. We work on Seerr when we have the time to do so, around our actual lives and other responsibilities. We're not obligated to drop everything we're doing to prioritize a feature simply because you feel entitled to have it.

If this feature is important enough to you that you don't want to wait for us to get to it, you're still welcome to contribute a PR. But repeatedly complaining that a free, volunteer-maintained project hasn't implemented your requested feature on your timeline isn't going to make it happen any faster.

**Comment by nicholas-shi** [comment]
Made a PR for it, I think the previous one tried adding a global server/user setting that is always applied. Kept it simple and just made it a toggle on the filters. Hopefully it passes muster 🤞
